rest_framework/fields.py

```diff
@@ -954,6 +956,10 @@ class FloatField(Field):
     def __init__(self, **kwargs):
         self.max_value = kwargs.pop('max_value', None)
         self.min_value = kwargs.pop('min_value', None)
+        self.localize = kwargs.pop('localize', False)
+        self.coerce_to_string = kwargs.pop('coerce_to_string', None)
+        if self.localize:
+            self.coerce_to_string = True
```

Comment:
Why do we need this here? 

In `to_representation`: 

```
        if self.localize:
             return number_format(value)
         if self.coerce_to_string:
             ...
```

So ` if self.localize`, `self.coerce_to_string` is never tested.

Reply:
Yes, because if localize is True, we always coerce to string (number_format returns a string). We only needs to check coerce_to_string if localize is False. (And we override coerce_to_string to True in the __init__, so when localize is True, coerce_to_string must be True).

Reply:
So we can drop this line...? (We don't need to override `coerce_to_string` since it will never be used.)

Reply:
It could be used if we don't set localize to True, we could have localize = False and coerce_to_string = True.

Reply:
Sure... Line 960 is fine. This comes into play in `to_representation` when `localize` is `False. 

But line 961 and 962 are no-ops really (right?) — So (I think) we can delete them. Yes/no?

Reply:
Ok got it... Yes it seems to...  just come from a copy & past from line 989 and 990 in DecimalField.__init__. I'll remove it.